src/libraries/block-snippets.json

```diff
@@ -1581,6 +1594,146 @@
+                                                "xs": "inline-block",
+                                                "md": "none"
+                                            }
+                                        }
+                                    },
+                                    "roles": null,
+                                    "hyperlink": {
+                                        "target": "_popup",
+                                        "targetKey": "popups/main-menu-popup"
+                                    }
+                                }
+                            ],
+                            "role": "article",
+                            "styles": {
+                                "instance": {
+                                    "grid-cell": {
+                                        "xs": {
+                                            "position": {
+                                                "col": 1,
+                                                "row": 1
+                                            },
+                                            "span": {
+                                                "cols": 1,
+                                                "rows": 1
+                                            },
+                                            "alignment": {
+                                                "vertical": "center",
+                                                "horizontal": "between"
+                                            }
+                                        },
+                                        "xl": {
+                                            "alignment": {
+                                                "vertical": "center",
+                                                "horizontal": "between"
+                                            }
+                                        },
+                                        "md": {
+                                            "alignment": {
+                                                "vertical": "center",
+                                                "horizontal": "between"
+                                            }
+                                        },
+                                        "lg": {
+                                            "alignment": {
+                                                "vertical": "center",
+                                                "horizontal": "between"
+                                            }
+                                        }
+                                    },
+                                    "padding": {
+                                        "xs": {
+                                            "top": 10,
+                                            "bottom": 10,
+                                            "left": 10,
+                                            "right": 10
```

Comment:
TBH I'm not familiar with this kind of UI definition, but could not be this simplify for something like: padding: { xs: 10 }?

Reply:
Yes, that would be simpler, but shortcuts not supported by the style compiler yet. Later on, we'll surely simplify it.